src/Routing.tsx

```
@@ -1,43 +1,37 @@
-
-import { Bullseye, Spinner } from '@patternfly/react-core';
-import { InvalidObject } from '@redhat-cloud-services/frontend-components/InvalidObject';
-
-import MigrationAssessmentPage from './pages/MigrationAssessmentPage';
-import MigrationWizardPage from './pages/MigrationWizardPage';
+import React, { Suspense } from "react";
+import { Route as RouterRoute, Routes as RouterRoutes } from "react-router-dom";
+import { InvalidObject } from "@redhat-cloud-services/frontend-components/InvalidObject";
+import { Bullseye, Spinner } from "@patternfly/react-core";
+import MigrationAssessmentPage from "./pages/MigrationAssessmentPage";
+import MigrationWizardPage from "./pages/MigrationWizardPage";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)